cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/web/OAuth20AccessTokenControllerTests.java

```diff
@@ -439,7 +438,7 @@ private void internalVerifyClientOK(final RegisteredService service, final boole
         mockRequest.setParameter(OAuthConstants.CODE, code.getId());
         final MockHttpServletResponse mockResponse = new MockHttpServletResponse();
         requiresAuthenticationInterceptor.preHandle(mockRequest, mockResponse, null);
-        oAuth20AccessTokenController.handleRequest(mockRequest, mockResponse);
+        oAuth20AccessTokenController.handleRequestInternal(mockRequest, mockResponse);
         assertNull(oAuth20AccessTokenController.getTicketRegistry().getTicket(code.getId()));
         assertEquals(200, mockResponse.getStatus());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['200' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2530574633/issues/source?bid=3256115&fileBranchId=3273836#l443)